### ISSUE TITLE
Consider showing video splash screen instead of first video frame

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#547]
Currently the video player simply shows the first frame of the video, which is usually black. We could instead show a splash screen (the same as the thumbnail) there. At least I assume Paella can be configured to do so. (And even if not, we could potentially do it ourselves even).

However, we should likely add a new database column for the url to the splash image, as it should have a higher resolution. Opencast does not currently provide any higher quality images than the one we already used for the thumbnail. Compare https://github.com/opencast/opencast/issues/3753. We could also add a generic column `images` with a json structure with potentially multiple images, giving us more flexibility in the future without having to change columns all the time. But not sure.

**Comment by geichelberger** [comment]
This is a duplicate of the issue https://github.com/elan-ev/tobira/issues/464

**Comment by JulianKniephoff** [comment]
True, but let me transfer some of the additional info in here over.
